### DISCUSSION TITLE
BYD砂金，三锤砸死我的通关梦

### DISCUSSION THREAD

**Discussion by Crafty-rabbit** [share121/inter-knot#1338] [其他]
为逝者哀苦
![d5ba5047d89c72c7fb65bcd32df97ab1](https://github.com/user-attachments/assets/e30ce07c-1e62-46c4-9749-914587f896db)
![d5ba5047d89c72c7fb65bcd32df97ab1](https://github.com/user-attachments/assets/60a6a6aa-b484-4006-9cb0-39c37dff9578)

**Comment by Crafty-rabbit** [comment]
![373352454903e22bb29da4b1c5611e6d](https://github.com/user-attachments/assets/b4807f22-cf46-44f8-ac22-eea95292ebbf)

**Comment by likeseele** [comment]
哥们把鸭鸭换成火主试试，应该就稳了，前期还是带双生存更稳吧
